packages/mantine/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@coveord/plasma-mantine",
-    "version": "47.9.3",
+    "version": "47.9.1",
```

Comment:
We should keep the latest version in the package.json I think

Reply:
oops 🙈  yes indeed